agreement/proposal.go

```diff
@@ -201,6 +201,7 @@ func verifyNewSeed(p unauthenticatedProposal, ledger LedgerReader) error {
 		if !ok {
 			return fmt.Errorf("payload seed proof malformed (%v, %v)", prevSeed, p.SeedProof)
 		}
+		_ = vrfOut // ignoring output of Verify
```

Comment:
I think that I'd rather seeing 
```golang
ok, _ := verifier.Verify(p.SeedProof, prevSeed)
```
instead if we're going to intentionally ignore the vrf output.